app/models/group.rb

```diff
@@ -19,7 +19,21 @@ class Group < ActiveRecord::Base
   has_many :leaders, -> { where(group_members: { leader: true }) },
            through: :group_members, source: :user
 
+  after_destroy :destroy_notifications
+
   def led_by?(user)
     leaders.include? user
   end
+
+  def notifications
+    Notification.where('uniqueid ilike ?', '%new_group%').select do |n|
```

Comment:
I'm not super familiar with the notifications, but it sounds like there is a json store in uniqueid that looks something like `comment_on_strategy_1`? 

Can we include the id in the query?
```query_string = "new_group_#{groupid}"
`Notification.where('uniqueid ilike ?', '%query_string%')```